src/test/scala/org/ergoplatform/nodeView/ErgoNodeViewHolderSpec.scala

```diff
@@ -368,7 +409,7 @@ class ErgoNodeViewHolderSpec extends ErgoPropertyTest with NodeViewTestOps with
     }
   }
 
-  val verifyingTxCases = List(t10, t11, t12, t13)
+  val verifyingTxCases = List(t11, t12, t13, t14)
```

Comment:
Maybe we should also move t7 and t8 here?

Reply:
done